pkg/combustion/templates/script-base.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# combustion: network
```

Comment:
Do we need this? I'm wondering whether it affects the network component but I can test this at a later point.

Reply:
We absolutely need this and is the big blocker that took so much of Danny's time. Without it, it just straight up doesn't work.